components/odh-notebook-controller/config/manager/manager.yaml

```diff
@@ -54,3 +54,9 @@ spec:
             requests:
               cpu: 500m
               memory: 256Mi
+          env:
+            - name: SET_PIPELINE_RBAC
+              valueFrom:
+                configMapKeyRef:
+                  name: config
+                  key: SET_PIPELINE_RBAC
```

Comment:
why are we using additional key, instead of simple env var  from code ?
we would only need this our code base side and not something that is required by the user

Reply:
Thanks for the review! I followed as example what is written for ISTIO env. I guess yes just having an 
```
env:
 - name: SET_PIPELINE_RBAC
   value: "false"
``` 
like that would do the trick. I will check this way